### PR TITLE
feat: create persona

### PR DESCRIPTION
**Pull request by lelenei**
closes #
dashboard 2.0 figma 1.5/1.6

### CODE REVIEW

File: packages/dashboard/src/locales/en.json [hunk 1]
```diff
@@ -79,5 +79,13 @@
     "settings_email_hosting_service_title": "Email Hosting Service",
     "settings_email_hosting_service_desc": "Unauthorized",
     "settings_log_out_title": "Log Out",
-    "settings_log_out_desc": "Please make sure your data is backed up and log out"
+    "settings_log_out_desc": "Please make sure your data is backed up and log out",
+
```
Comment (septs) [inline packages/dashboard/src/locales/en.json:83]: remove the line

File: packages/dashboard/src/pages/Register/Components/ContainerPage.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    goback?: React.ReactElement
+}
+
+export const ContainerPage = memo<ContainerPageProps>(({ children, goback }) => {
+    const classes = useStyles()
+    const t = useDashboardI18N()
+    const Navigate = useNavigate()
+
+    return (
+        <div className={classes.container}>
```
Comment (septs) [inline packages/dashboard/src/pages/Register/Components/ContainerPage.tsx:77]: why use `<div>` tag.

**Comment by septs** [comment]
plz resolve conflict on develop branch

**Comment by nuanyang233** [comment]
Hi, The new register design does not include 1.5/1.6.

**Pull request closed without merging**